dev/examples/controls/line.js

```diff
@@ -31,7 +31,7 @@ const regions = [
 export const renderLine = (id) => {
     const lineDefault = Carbon.api.graph(getDemoData(`#${id}`, "LINE_DEFAULT"));
     lineDefault.loadContent(
-        Carbon.api.line(getDemoData(`#${id}`, "LINE_DEFAULT").data[0])
+        Carbon.api.line(getDemoData(`#${id}`, "LINE_DEFAULT").data[2])
```

Comment:
May i know the reason why did you changed the content in here. i guess it is changing the shape of our basic line graph.

![image](https://user-images.githubusercontent.com/63670637/87202598-7c1fe680-c2c6-11ea-8f97-1ab268783b59.png)

Reply:
I had changed for some testing purpose. I reverted now.
https://github.com/SharathAlur/carbon-graphs/commit/788fde01aa11c505624e86e5a4ce9bc7da6d0813#diff-8ef55c17e956a86bd177242c976fc979L34